docs/website/docs/user-guides/advanced/connecting-a-service.md

```diff
@@ -0,0 +1,229 @@
+---
+title: Connecting to a service
+sidebar_position: 1
+---
+
+This tutorial will show you how you can connect your Go application to a mongodb service.
```

Comment:
Avoid sentences like these, we already know it's a tutorial.

Instead, add a quick sentence explaining what a service is?

Ex:

`Building on top of the [Go quickstart guide](../quickstart/go.md), we will extend the application by connecting it to a deployable MongoDB service.`